website/docs/r/vapp_firewall_rules.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_firewall_rules"
+sidebar_current: "docs-vcd-resource-vapp-firewall-rules"
+description: |-
+  Provides a vCloud Director vApp Firewall resource. This can be used to create, modify, and delete firewall settings and rules.
+---
+
+# vcd\_vapp\_firewall\_rules
+
+Provides a vCloud Director vApp Firewall resource. This can be used to create,
+modify, and delete firewall settings and rules.
```

Comment:
```suggestion
modify, and delete firewall settings and rules in a vApp network.
```
Also, please make a link out of "vApp network" and point to the documentation page of it, so we link these two pages between each other.

Reply:
added